src/service/mercury/index.ts

```diff
@@ -422,8 +422,14 @@ export class MercuryClient {
   tokenDetails = async (
     pubKey: string,
     contractId: string,
-    network: NetworkNames
-  ): Promise<{ name: string; symbol: string; decimals: string }> => {
+    network: NetworkNames,
+    fetchBalance: boolean = false,
```

Comment:
NIT: can we use a boolean naming convention here? `should_fetch_balance` or something like that

Reply:
https://github.com/stellar/freighter-backend/pull/204/commits/93bfa7367f9d71f3e6757027b72c300707f39e42